internal/cmd/setup/frontend.go

```diff
@@ -12,12 +12,12 @@ func yarnCheck(app meta.App) error {
 		return err
 	}
 	if _, err := exec.LookPath("yarnpkg"); err != nil {
-		err := run(exec.Command("npm", "install", "-g", "yarn"))
+		err := run(exec.Command("npm", "install", "-g", "yarn@berry"))
```

Comment:
This code is for the `buffalo setup` command which is used not only for a newly created app but also for an existing app such as just checked out from its repository to a new dev environment. In the latter case, we cannot guarantee the app is configured for berry and I guess this approach is not the best solution. I think we can keep the original line here. Do you check such a scenario?

Reply:
That sounds good, will revert.